### PR TITLE
Feature/uttmm 77

### PR DESCRIPTION
**Pull request by PanasiukAndriy**

### CODE REVIEW

File: TheraLang.Web/ClientApp/src/app/shared/api-endpoint.constants.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 export const liqpayCheckoutUrl = "https://www.liqpay.ua/api/3/checkout";
-export const baseUrl = "/api/";
+export const baseUrl = "http://localhost:5000/api/";
```
Comment (AlexBoichuk) [inline TheraLang.Web/ClientApp/src/app/shared/api-endpoint.constants.ts]: Delete localhost

File: TheraLang.Web/ClientApp/src/app/add-resources-to-project/add-resources-to-project.component.ts [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    // let var1 = new Resource();
+    // var1.id = 1;
+    // var1.name = 'some text1';
+    // let var2 = new Resource();
+    // var2.id = 2;
+    // var2.name = 'some text2';
+    // let var3 = new Resource();
+    // var3.id = 3;
+    // var3.name = 'some text3';
+    // this.resources = new Array(var1, var2, var3);
```
Comment (AlexBoichuk) [inline TheraLang.Web/ClientApp/src/app/add-resources-to-project/add-resources-to-project.component.ts]: delete it

File: TheraLang.Web/ClientApp/src/app/add-resources-to-project/add-resources-to-project.component.ts [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    this.resources = await this.service.getAllResourcess()
+
+    this.filteredResources = this.resources;
+    debugger
```
Comment (AlexBoichuk) [inline TheraLang.Web/ClientApp/src/app/add-resources-to-project/add-resources-to-project.component.ts]: Delete it

File: TheraLang.Web/ClientApp/src/app/add-resources-to-project/add-resources-to-project.component.ts [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+  }
+
+  Add(_resourceId: number): Observable<any> {
+    debugger;
```
Comment (AlexBoichuk) [inline TheraLang.Web/ClientApp/src/app/add-resources-to-project/add-resources-to-project.component.ts]: Delete it

File: TheraLang.Web/ClientApp/src/app/add-resources-to-project/add-resources-to-project.component.ts [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+    const newResourceToProject = new ResourceToProject();
+    newResourceToProject.resourceId = _resourceId;
+    newResourceToProject.projectId = this.data.id;
+    debugger;
```
Comment (AlexBoichuk) [inline TheraLang.Web/ClientApp/src/app/add-resources-to-project/add-resources-to-project.component.ts]: Delete it

File: TheraLang.Web/ClientApp/src/app/shared/api-endpoint.constants.ts (hunk 1 above)
Comment (dgzargo) [inline TheraLang.Web/ClientApp/src/app/shared/api-endpoint.constants.ts]:
```suggestion
export const baseUrl = "/api/";
```

File: TheraLang.Web/ClientApp/tsconfig.app.json [hunk 6]
```diff
@@ -1,7 +1,7 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
-    "outDir": "./dist/out-tsc/app",
+    "outDir": "./out-tsc/app",
```
Comment (dgzargo) [inline TheraLang.Web/ClientApp/tsconfig.app.json]: revert change

File: TheraLang.Web/ClientApp/tsconfig.json [hunk 7]
```diff
@@ -2,7 +2,7 @@
   "compileOnSave": false,
   "compilerOptions": {
     "baseUrl": "./",
-    "outDir": "./dist/out-tsc",
+    "outDir": "./out-tsc",
```
Comment (dgzargo) [inline TheraLang.Web/ClientApp/tsconfig.json]: revert change

File: TheraLang.Web/Controllers/HomeController.cs [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+        public IActionResult Index()
+        {
+            return View();
+        }
+        public ActionResult Routing() // Routing  
+        {
+            return View();
+        }
+    }
+}
```
Comment (dgzargo) [inline TheraLang.Web/Controllers/HomeController.cs]: delete file

File: TheraLang.Web/Startup.cs [hunk 9]
```diff
@@ -34,18 +34,19 @@ public void ConfigureServices(IServiceCollection services)
...
             services.AddSpaStaticFiles(configuration =>
             {
                 configuration.RootPath = "ClientApp/dist";
             });
 
+            services.AddMvc()
+                .AddPiranhaManagerOptions()
+                .SetCompatibilityVersion(CompatibilityVersion.Version_2_2)
+                .AddFluentValidation(fv => { fv.RunDefaultMvcValidationAfterFluentValidationExecutes = false; });
+
```
Comment (dgzargo) [inline TheraLang.Web/Startup.cs]: switch places

File: TheraLang.DLL/Services/IResourceService.cs [hunk 10]
```diff
@@ -23,5 +23,12 @@ public interface IResourceService
         IEnumerable<Resource> GetResourcesByCategoryId(int categoryId, int pageNumber, int recordsPerPage);
 
         IEnumerable<Resource> GetAllResourcesByProjectId(int projectId);
+
+        Task AddResourceToProject(ResourceProject resourceProject, int userId);
```
Comment (dgzargo) [inline TheraLang.DLL/Services/IResourceService.cs]: use GUID or don't use it at all either

**Pull request closed without merging**